packages/flutter/lib/src/rendering/paragraph.dart

```diff
@@ -174,6 +176,15 @@ class RenderParagraph extends RenderBox {
     markNeedsLayout();
   }
 
+  ui.Locale get locale => _textPainter.locale;
```

Comment:
oh, also, if Locale is part of your API, you should reexport it, so that users of your API don't have to import dart:ui